hopli/src/network_registry.rs

```diff
@@ -284,9 +288,9 @@ impl NetworkRegistrySubcommands {
         eligibility: Option<bool>,
     ) -> Result<(), HelperErrors> {
         // read all the safe addresses
-        let mut safe_eth_addresses: Vec<H160> = Vec::new();
+        let mut safe_eth_addresses: Vec<Address> = Vec::new();
         if let Some(addresses) = safe_address {
-            safe_eth_addresses.extend(addresses.split(',').map(|addr| H160::from_str(addr).unwrap()));
+            safe_eth_addresses.extend(addresses.split(',').map(|addr| Address::from_str(addr).unwrap()));
```

Comment:
Also here, use `?` instead of `unwrap` since the function returns `Result`

Reply:
changed